src/i18n.ts

```diff
@@ -0,0 +1,23 @@
+function loadResourcesFromFile() {
+  const lang = window.i18next.language;
```

Comment:
Rather than directly referencing `window.i18next`, I'd mark `i18next` as a peer dependency and import it in this file.

Reply:
Getting undefined with `i18next` reference. Do you know what might be the reason?

Reply:
When in the app is that happening? It's vaguely possible that `language` is `undefined`, but `window.i18next` should be set [here](https://github.com/openmrs/openmrs-esm-core/blob/eeb48a4c573111d7323956164d75e7f64723543b/packages/shell/esm-app-shell/src/locale.ts#L14). That's called _after_ apps are registered, but it should be before anything gets mounted (since single-spa isn't called until `setupI18n()` succeeds).

Reply:
<img width="778" alt="image" src="https://github.com/openmrs/openmrs-form-engine-lib/assets/51502471/1ce97749-3db4-4238-bfd7-4526d2f12d98">
<img width="504" alt="image" src="https://github.com/openmrs/openmrs-form-engine-lib/assets/51502471/08922918-adee-409a-924c-4372e8cf8fc8">